__tests__/adaptiveCards.js

```diff
@@ -58,7 +55,17 @@ test('action styles', async () => {
 });
 
 test('breakfast card with custom host config', async () => {
-  const adaptiveCardHostConfig = createAdaptiveCardsHostConfig({ bubbleTextColor: '#FF0000' });
+  const adaptiveCardHostConfig = {
+    containerStyles: {
```

Comment:
Built a new Adaptive Cards host config, which is no longer inherited from Web Chat built-in host config. So we need to update the screenshot as well.